File: 06Adapter/Students/2018/DuchowskaKatarzyna/zad1/Seacraft.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Adapter.zad1
{
    //public class Seacraft : ISeacraft
    public class Seacraft : Aircraft, ISeacraft
    {
        int speed = 0;

        public virtual void IncreaseRevs()
        {
            speed += 10;
            Console.WriteLine("Seacraft engine increases revs to " + speed + " knots");
        }
        public int Speed
        {
            get { return speed; }
        }
    }
}

File: 06Adapter/Students/2019/AliaksandrNovik/src/main/java/twoWayAdapter/Seacraft.java
package twoWayAdapter;

public class Seacraft implements ISeacraft {
	private int speed = 0;

	@Override
	public void increaseRevs() {
		speed += 10;
		System.out.println("Seacraft engine increases revs to " + speed + " knots");
	}

	public int speed() {
		return speed;
	}
}